### PR TITLE
Add para calling out app-cloud integration to dashboard runs page

### LINKED ISSUE #4704: Highlight benefits of ACI in Dashboard section
### Subject

Dashboard

### Description

Add something to Dashboard docs calling out the integration of Dashboard data into the Cypress App.

### PR DESCRIPTION
**Pull request by debrisapron**
Closes #4704

### CODE REVIEW

**Comment by debrisapron** [comment]
New paragraph here: https://cypress-documentation-pea2w9eg7-cypress-io.vercel.app/guides/dashboard/runs#Cypress-App

File: content/guides/dashboard/runs.md [hunk 1]
```diff
@@ -77,6 +77,17 @@ spec. You can also download the video.
 
 <DocsImage src="/img/dashboard/videos-of-recorded-test-run.png" alt="Video of test runs" ></DocsImage>
 
+### Cypress App
+
+For even more visibility into your test results, information about recent runs
+is surfaced in the [Cypress App](/guides/core-concepts/cypress-app). Aggregate
```
Comment (pstakoun) [inline content/guides/dashboard/runs.md:83]: @mattvCypress should this be "Cypress app"?

File: content/guides/dashboard/runs.md [hunk 2]
```diff
@@ -77,6 +77,17 @@ spec. You can also download the video.
...
+results of your latest runs are shown in
+[The Spec Explorer](/guides/core-concepts/cypress-app#The-Spec-Explorer) while
```
Comment (pstakoun) [inline content/guides/dashboard/runs.md:85]: @mattvCypress capitalization here ok?

File: content/guides/dashboard/runs.md (hunk 1 above)
Comment (mattvCypress) [reply content/guides/dashboard/runs.md:83]: @pstakoun The link should be "Cypress app". The header appears to be title cased so "Cypress App" (row 80) is technically fine there.

File: content/guides/dashboard/runs.md (hunk 2 above)
Comment (mattvCypress) [reply content/guides/dashboard/runs.md:85]: Yes

**Review by pstakoun — APPROVED** [review]

**Pull request merged**